### PR TITLE
Refactoring App.js

### PR DESCRIPTION
**Pull request by MechanicKim**
Refactoring!

Devided component - component and props
For legibility and maintenance

### CODE REVIEW

File: src/utils/UrlMeta.js [hunk 1]
```diff
@@ -12,7 +13,7 @@ class UrlMeta {
   static API_MENTOR_RESP = 'match/response/';
   static API_EDIT_GENERAL = 'users/editGeneral/';
   static API_LOCAL_SIGNUP = 'users/localSignUp/';
-  static API_LOCAL_SIGNIN = 'users/localSignIn/'
+  static API_LOCAL_SIGNIN = 'users/localSignIn/';
```
Comment (heartjigi) [inline src/utils/UrlMeta.js]: This js file dont use anymore

**Comment by heartjigi** [comment]
It's coool~!

**Comment by MechanicKim** [comment]
Oh! Have a nice flight! And say hello to three guys kkk

2016-11-16 10:28 GMT+09:00 heartjigi notifications@github.com:

**Comment by SarahhhC** [comment]
LGTM!! Thanks @MechanicKim :

**Pull request merged**